include/proj/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(installdest ${INCLUDEDIR}/proj)
+set(installdest ${CMAKE_INSTALL_INCLUDEDIR}/proj)
```

Comment:
A minor nit is to remove this line (`installdest` is used once), and use `DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/proj` below.